tests/unit/services/ajax-test.js

```diff
@@ -134,6 +134,49 @@ test("request() promise label is correct", function(assert) {
   assert.equal(postPromise._label, 'ember-ajax: POST to /posts');
 });
 
+test("post() promise label is correct", function(assert) {
+  service = Service.create();
+  const url = '/posts';
```

Comment:
let/const/var describing the variable binding, not the content.

const is immutable binding, let is block scoped binding, var is yolo binding.